frontend/src/types.ts

```diff
@@ -311,3 +311,12 @@ export interface PluginErrorType {
     name?: string
     event?: Record<string, any>
 }
+
+export interface PluginConfigSchemaType {
+    hint: string
+    key: string
+    name: string
+    order: number
+    required: boolean
+    type: string
+}
```

Comment:
This is a shared type that you can just get with

```
import { PluginConfigSchema } from '@posthog/plugin-scaffold
```

... so we shouldn't re-define it here